docs/Benchmark.mdx

```diff
@@ -1,13 +1,15 @@
 ---
 id: benchmark
-title: Benchmarks
+title: Benchmark
```

Comment:
The reason for this problem is that I simply merged #2 . And VS Code doesn't show the diff between Benchmark.md and Benchmark.mdx. I have installed GitLens plugin, such problems will be solved in the next commit.

Reply:
@Selflocking don't just copy everything from: https://github.com/casbin/casbin-website-v2/pull/2 to this PR. Judge it first and make sure only valuable things go to this one.